enricher/standard/src/test/java/io/fabric8/maven/enricher/standard/DefaultServiceEnricherTest.java

```diff
@@ -35,6 +35,8 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import javax.sound.midi.SysexMessage;
```

Comment:
Doesnt look right

Reply:
I'm helping devang with tests and have marked this PR as WIP. I asked him to update his branch. Will ping whenever it gets ready(in few hours ;-) )